### ISSUE TITLE
Feedback on this program

### ISSUE THREAD

**Issue by MikillRosen** [Aldrin-John-Olaer-Manalansan/RA2YR-reMIXer#6]
Since this is somewhat new and is supposed to be a successor to "RA2 MIX Unprotector Master", I actually found it to be a slightly worse version of its predecessor. There's less user control in the new program. It forces an input and output directory system, it automatically opens the input folder on launch, opens the output directory on use, doesn't allow me to add file extensions to recognize unlike in the previous version (some mods change the file extensions of their MIX files to further their encryption), and it doesn't even show the MIX files by default, having me to manually refresh. I don't get the point of this overhaul, you ended up making the program worse without any noticeable upgrade to the decryption process. The program fights against my control and I don't like it. Just go back to the previous version and improve from there.

**Comment by Aldrin-John-Olaer-Manalansan** [comment]
Version 3 will be out soon and your pointed out critics has been revised already. Will tag this ticket once it gets released marked for UAT.

**Comment by MikillRosen** [comment]
UAT? Could you tell me what that means?

**Comment by Aldrin-John-Olaer-Manalansan** [comment]
[User Acceptance Test](https://www.panaya.com/blog/testing/what-is-uat-testing/#:~:text=User%20Acceptance%20Testing%20(UAT)%2C,do%20in%20real%2Dworld%20situations.) - After the app gets released here at github. All tickets marked as solved by that version will be monitored for user feedbacks. Once majority of tickets proves this to new version to be effective. Then this version will then be announced to all platforms that currently distributes this tool.

In simple terms, once the app gets a new version containing this ticket's solution, you'll be asked to test it out and provide feedback afterwards.

**Comment by Aldrin-John-Olaer-Manalansan** [comment]
This ticket is now marked for UAT for 1 week. Test the latest build and report here if a problem connected to this ticket have been encountered.

**Comment by MikillRosen** [comment]
This update is certainly an improvement, but there are still some things I'd like to point out.

Entire window cannot be resized.
Dragging and dropping files onto the window should be added.
The program saves decrypted files as new files. I'd prefer this to be optional and have overwriting as an option.
A neat detail in RA2YR MIX Unprotector Master was that it showed the original header and the unprotected header of MIX files.

**Comment by Aldrin-John-Olaer-Manalansan** [comment]
This topic is now assumed to have been satisfied by the current tool's capabilities.

Ticket closed.
